File: MODUL2/PYTHON/PRAK205_2410817310002_MuhammadRakhaAthallah.py
A,B = map(float, input().split())

C = (B * B - A * A) ** 0.5
keliling = A + B + C
luas = (A * C) / 2

print(f"Alas = {int(C)} cm") 
print(f"Tinggi = {int(A)} cm") 
print(f"Keliling = {int(keliling)} cm") 
print(f"Luas = {int(luas)} cm^2") 

A1 = float(input())
B1= float(input())

C1 = (B1 * B1 - A1 * A1) ** 0.5
keliling1 = A1 + B1 + C1
luas1 = (A1 * C1) / 2

print(f"Alas = {int(C1)} cm") 
print(f"Tinggi = {int(A1)} cm") 
print(f"Keliling = {int(keliling1)} cm") 
print(f"Luas = {int(luas1)} cm^2")

File: MODUL2/C/PRAK205_2410817310002_MuhammadRakhaAthallah.c
#include <stdio.h>
#include <math.h>

int main(){
    //Variabel pertama
    float A, B, C, keliling, Luas;

    scanf("%f", &A);
    scanf("%f", &B);

    C = sqrt(B * B - A * A);
    keliling = A + B + C;
    Luas = (A * C) / 2;

    printf("Alas = %d cm\n", (int)C);
    printf("Tinggi = %d cm\n", (int)A);
    printf("Keliling = %d cm\n", (int)keliling);
    printf("Luas : %d cm^2\n", (int)Luas);

    //variabel kedua
     float A1, B1, C1, keliling1, Luas1;

     scanf("%f", &A1);
    scanf("%f", &B1);

    C1 = sqrt(B1 * B1 - A1 * A1);
    keliling1 = A1 + B1 + C1;
    Luas1 = (A1 * C1) / 2;

    printf("Alas = %d cm\n", (int)C1);
    printf("Tinggi = %d cm\n", (int)A1);
    printf("Keliling = %d cm\n", (int)keliling1);
    printf("Luas : %d cm^2\n", (int)Luas1);

    return 0;
}